ipaserver/plugins/cert.py

```diff
@@ -793,6 +838,9 @@ def execute(self, csr, all=False, raw=False, **kw):
                 api.Command['host_mod'](principal.hostname, **kwargs)
             elif principal_type == USER:
                 api.Command['user_mod'](principal.username, **kwargs)
+            elif principal_type == KRBTGT:
+                root_logger.error("Profiles used to store cert should't be "
```

Comment:
When I tried root_logger was the only way to use it IIRC.
I do not think this calss has a self.log